kube-deploy/group_vars/all.yml

```diff
@@ -18,6 +18,8 @@ bootstrap_os: ubuntu
 public_iface: enp0s3
 nat_iface: enp0s3
 proxy_enable: false
+#Only enable this if you are deploying on ARM or ARM64 with Fannel SDN provider
+arm_enabled: false
```

Comment:
Looking at the setup module output on my arm64 boards, there are two ansible variables we could use.  Alternatively, we could pull it directly from the board using `uname -m`.
```
"ansible_machine": "aarch64",
"ansible_architecture": "aarch64",
```

To map these architecture strings to the names used in actual docker image, take a look at the case statement in https://github.com/kubernetes/kubernetes/blob/85d328f8da12f0f0e5b7e6dd2356e56cd584fc3e/hack/local-up-cluster.sh#L234

We could exclude `s390x` and `x86` because google doesn't even release images for those.  https://console.cloud.google.com/kubernetes/images/list?location=GLOBAL&project=google-containers  CTRL+F for a common k8s image like "kube-apiserver" and note the architectures kubernetes supports.

As for needing test instances, I can try to put up a couple instances on the internet.  Alternatively, scaleway has pretty cheap arm64 VPS instances at $2.99/month.